app/Libraries/Policies/Policy.php

```diff
@@ -70,7 +70,7 @@ public function deny(string $message, int $status = 403)
      */
     public function withPolicy(PolicyInterface $policy): self
     {
-        $this->policies[get_class($policy)] = $policy;
+        $this->policies[$policy::class] = $policy;
```

Comment:
Interesting. I would have never thought to do it this way. Neat.

Reply:
Me either.